src/globalInfo/dxEnvVars.ts

```diff
@@ -0,0 +1,394 @@
+    description: getMessage(EnvironmentVariable.SF_AUTOUPDATE_DISABLE),
+    synonymOf: null,
+  },
+  [EnvironmentVariable.SF_DISABLE_SOURCE_MEMBER_POLLING]: {
+    description: getMessage(EnvironmentVariable.SF_DISABLE_SOURCE_MEMBER_POLLING),
+    synonymOf: null,
+  },
+  [EnvironmentVariable.SF_DISABLE_TELEMETRY]: {
+    description: getMessage(EnvironmentVariable.SF_DISABLE_TELEMETRY),
+    synonymOf: null,
+  },
+  [EnvironmentVariable.SF_DNS_TIMEOUT]: {
+    description: getMessage(EnvironmentVariable.SF_DNS_TIMEOUT),
+    synonymOf: null,
+  },
+  [EnvironmentVariable.SF_DOMAIN_RETRY]: {
+    description: getMessage(EnvironmentVariable.SF_DOMAIN_RETRY),
+    synonymOf: null,
+  },
+  [EnvironmentVariable.SF_IMPROVED_CODE_COVERAGE]: {
+    description: getMessage(EnvironmentVariable.SF_IMPROVED_CODE_COVERAGE),
+    synonymOf: null,
+  },
+  [EnvironmentVariable.SF_INSTANCE_URL]: {
+    description: getMessage(EnvironmentVariable.SF_INSTANCE_URL),
+    synonymOf: null,
+  },
+  [EnvironmentVariable.SF_JSON_TO_STDOUT]: {
+    description: getMessage(EnvironmentVariable.SF_JSON_TO_STDOUT),
+    synonymOf: null,
+  },
+  [EnvironmentVariable.SF_LOG_LEVEL]: {
+    description: getMessage(EnvironmentVariable.SF_LOG_LEVEL),
+    synonymOf: null,
+  },
+  [EnvironmentVariable.SF_MAX_QUERY_LIMIT]: {
+    description: getMessage(EnvironmentVariable.SF_MAX_QUERY_LIMIT),
+    synonymOf: null,
+  },
+  [EnvironmentVariable.SF_MDAPI_TEMP_DIR]: {
+    description: getMessage(EnvironmentVariable.SF_MDAPI_TEMP_DIR),
+    synonymOf: null,
+  },
+  [EnvironmentVariable.SF_NPM_REGISTRY]: {
+    description: getMessage(EnvironmentVariable.SF_NPM_REGISTRY),
+    synonymOf: null,
+  },
+  [EnvironmentVariable.SF_PRECOMPILE_ENABLE]: {
+    description: getMessage(EnvironmentVariable.SF_PRECOMPILE_ENABLE),
+    synonymOf: null,
+  },
+  [EnvironmentVariable.SF_PROJECT_AUTOUPDATE_DISABLE_FOR_PACKAGE_CREATE]: {
+    description: getMessage(EnvironmentVariable.SF_PROJECT_AUTOUPDATE_DISABLE_FOR_PACKAGE_CREATE),
+    synonymOf: null,
+  },
+  [EnvironmentVariable.SF_PROJECT_AUTOUPDATE_DISABLE_FOR_PACKAGE_VERSION_CREATE]: {
+    description: messages.getMessage(
+      camelCase(EnvironmentVariable.SF_PROJECT_AUTOUPDATE_DISABLE_FOR_PACKAGE_VERSION_CREATE)
+    ),
+    synonymOf: null,
+  },
+  [EnvironmentVariable.SF_REST_DEPLOY]: {
+    description: getMessage(EnvironmentVariable.SF_REST_DEPLOY),
+    synonymOf: null,
+  },
+  [EnvironmentVariable.SF_SOURCE_MEMBER_POLLING_TIMEOUT]: {
+    description: getMessage(EnvironmentVariable.SF_SOURCE_MEMBER_POLLING_TIMEOUT),
+    synonymOf: null,
+  },
+  [EnvironmentVariable.SF_USE_GENERIC_UNIX_KEYCHAIN]: {
+    description: getMessage(EnvironmentVariable.SF_USE_GENERIC_UNIX_KEYCHAIN),
+    synonymOf: null,
+  },
+  [EnvironmentVariable.SF_USE_PROGRESS_BAR]: {
+    description: getMessage(EnvironmentVariable.SF_USE_PROGRESS_BAR),
+    synonymOf: null,
+  },
+};
+
+export class EnvVars extends Env {
```

Comment:
This class and methods should probably live in an `envVarAccessor` to maintain the pattern we're using with [aliases](https://github.com/forcedotcom/sfdx-core/blob/v3/src/globalInfo/accessors/aliasAccessor.ts), [orgs](https://github.com/forcedotcom/sfdx-core/blob/v3/src/globalInfo/accessors/orgAccessor.ts), and [tokens](https://github.com/forcedotcom/sfdx-core/blob/v3/src/globalInfo/accessors/tokenAccessor.ts).

Reply:
I was wondering where the best place to keep this class. I don't feel that it qualifies as an accessor and since it is closely tied to config aggregator, I moved the class to config.